src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java

```diff
@@ -3933,7 +3933,7 @@ private void drawMiningPage(int mouseX, int mouseY, float partialTicks) {
 				EnumChatFormatting.GRAY + "Ore.",
 				"",
 				EnumChatFormatting.GRAY + "Cost",
-				EnumChatFormatting.DARK_GREEN + "" + (int) Math.pow(tittyInsane + 2, 3.1) + " Mithril Powder"
+				EnumChatFormatting.DARK_GREEN + "" + NumberFormat.getInstance().format((int) Math.pow(miningSpeed + 2, 3)) + " Mithril Powder"
```

Comment:
why you change variable from titty to minignspeed

Reply:
ctrl c ctrl v moment

Reply:
i didnt see that one i am blind i just copy pasted i didnt really look at it before pr'ing